test/python/importer/jit_ir/node_import/debug-info.py

```diff
@@ -19,9 +19,14 @@
 def add3(t0, t1, t2):
   # TODO: Checks for debug info are quite hard with the new trailing debug
   # attribute print. See if this can be improved.
```

Comment:
I know this predates your change, but "this can be improved" with `mb.module.operation.print(enable_debug_info=True, use_local_scope=True)`
```python
import mlir.ir
with mlir.ir.Context() as c:
    c.allow_unregistered_dialects = True
    m = mlir.ir.Module.parse("\"test.op\"() {} : () -> () loc(\"loc\")")
    m.operation.print(enable_debug_info=True)
    print("// -----------------")
    m.operation.print(enable_debug_info=True, use_local_scope=True)
```
```mlir 
module  {
  "test.op"() : () -> () loc(#loc1)
} loc(#loc0)
#loc0 = loc("-":0:0)
#loc1 = loc("loc")
// -----------------
module  {
  "test.op"() : () -> () loc("loc")
} loc("-":0:0)
```

Reply:
Thanks! Test was updated.